CME/dataobjects/CMECredit.php

```diff
@@ -63,6 +63,16 @@ public function getFormattedHours()
 		return static::formatCreditHours($this->hours);
 	}
 
+	// }}}
+	// {{{ public function hasQuiz()
+
+	public function hasQuiz()
+	{
+		return $this->getInternalValue('quiz') !== null &&
+			$this->quiz instanceof CMEQuiz &&
+			count($this->quiz->question_bindings) > 0;
```

Comment:
I think the only potential downside here is extra database queries. At least on EM:RAP, this isn't an issue even on the huge account page that loads tons of credits or the certificate page.

Reply:
I agree, it seems like it would be ok to do it like this